SPM Example/PortalSwift/Portal.swift

```diff
@@ -102,21 +103,25 @@ class PortalWrapper {
     case cantLoadInfoPlist
   }
 
-  func registerPortal(apiKey: String, backup: BackupOptions, chainId: Int = 5, optimized: Bool = false, completion: @escaping (Result<Bool>) -> Void) {
+  func registerPortal(apiKey: String, backup: BackupOptions, chainId: Int = 11_155_111, optimized: Bool = false, completion: @escaping (Result<Bool>) -> Void) {
     do {
-      guard let ALCHEMY_API_KEY: String = Bundle.main.infoDictionary?["ALCHEMY_API_KEY"] as? String else {
+      guard let infoDictionary: [String: Any] = Bundle.main.infoDictionary else {
+        return completion(Result(error: PortalWrapperError.cantLoadInfoPlist))
+      }
+      guard let ALCHEMY_API_KEY: String = infoDictionary["ALCHEMY_API_KEY"] as? String else {
         print("Error: Do you have `ALCHEMY_API_KEY=$(ALCHEMY_API_KEY)` in your info.plist?")
         return
       }
       let keychain = PortalKeychain()
+
       // Configure the chain.
       self.portal = try Portal(
         apiKey: apiKey,
         backup: backup,
         chainId: chainId,
         keychain: keychain,
         gatewayConfig: [
-          11_155_111: "https://eth-sepolia.g.alchemy.com/v2/\(ALCHEMY_API_KEY)", 5: "https://eth-goerli.g.alchemy.com/v2/\(ALCHEMY_API_KEY)", 1: "https://eth-mainnet.g.alchemy.com/v2/\(ALCHEMY_API_KEY)", 80001: "https://polygon-mumbai.g.alchemy.com/v2/\(ALCHEMY_API_KEY)", 137: "https://polygon-mainnet.g.alchemy.com/v2/\(ALCHEMY_API_KEY)",
```

Comment:
let's not remove goerli. It's still helpful for many tests that aren't supported by sepolia yet.

Reply:
Will add that back! I don't think that was my code removing that but maybe it was 🤷 Not sure why I did that